docs/plugins/backend-plugin.md

```diff
@@ -72,38 +72,19 @@ to your backend.
 yarn --cwd packages/backend add @internal/plugin-carmen-backend@^0.1.0 # Change this to match the plugin's package.json
```

Comment:
I note that the above `yarn new` command actually already adds it to `packages/backend`. But ... why doesn't it do that with a `workspace:^` version? 🤔

Reply:
Hmm, yeah, this should definitely be `workspace:^`, let's remove this line during the rewrite?